drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/rightpanel/SubDockView.java

```diff
@@ -44,5 +44,9 @@
          */
```

Comment:
@yesamer 
Pleas declare the _reset method also in the _view_ interface (see above comments)

Reply:
Done